sync/class.jetpack-sync-module-users.php

```diff
@@ -226,7 +250,12 @@ function save_user_cap_handler( $meta_id, $user_id, $meta_key, $capabilities ) {
 			return;
 		}
 
-		$user =  get_user_by( 'id', $user_id );
+		//The jetpack_sync_register_user payload is identical to jetpack_sync_save_user, don't send both
+		if ( $this->is_create_user() ) {
```

Comment:
Moved, though it's not obvious which is more performant (unless the user is on php 5.5+)